frontend/src/mixins/shootItem.js

```diff
@@ -54,13 +55,17 @@ export const shootItem = {
       return get(this.shootMetadata, 'annotations', {})
     },
     shootGardenOperation () {
-      return this.shootAnnotations['shoot.garden.sapcloud.io/operation']
+      return this.shootAnnotations['gardener.cloud/operation']
     },
     shootPurpose () {
       return this.shootAnnotations['garden.sapcloud.io/purpose']
     },
     shootExpirationTimestamp () {
-      return this.shootAnnotations['shoot.garden.sapcloud.io/expirationTimestamp']
+      var expirationTimestamp = this.shootAnnotations['shoot.gardener.cloud/expiration-timestamp']
```

Comment:
`return this.shootAnnotations['shoot.gardener.cloud/expiration-timestamp'] || this.shootAnnotations['shoot.garden.sapcloud.io/expirationTimestamp']`